tests/e2e/docs/vmc_tkg_setup.md

```diff
@@ -68,6 +68,8 @@ The section outlines how to set the env variable for running the e2e test.
     export SHARED_NFS_DATASTORE_URL="<shared-NFS-datastore-url>"
     #shared VMFS datastore url
     export SHARED_VMFS_DATASTORE_URL="<shared-VMFS-datastore-url>"
+    #management datastore url
```

Comment:
mention that this is required for VMC test bed only

Reply:
Its mentioned in vmc_tkg_setup.md, should we explicitly mention that?

Reply:
Is this variable used only for VMC TKG setup or can be used for for VMC vanilla as well

Reply:
The .md file is specific for VMC TKG and the test case is expected to run on the VMC TKG testbed, for VMC Vanilla we don't have new automation.